src/page/generate/__test__/GenerateDocumentPage.test.tsx

```
@@ -1,231 +1,90 @@
-    supabase: {
-        from: vi.fn().mockReturnThis(),
-        insert: vi.fn().mockReturnThis()
-    }
+import { describe, it, expect, vi } from 'vitest';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router';
+import GenerateDocumentPage from '../GenerateDocumentPage';
+
+// Mock the components used within GenerateDocumentPage
+vi.mock('@/components/ui/generate-form', () => ({
+    GenerateDocumentSteps: ({ steps, title }: { steps: any[], title: string }) => (
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)